src/kvstore/transaction_impl.cc

```diff
@@ -516,3 +516,21 @@ void TransactionImpl::Commit()
   bool committed = db_->CommitResult(pos);
   assert(committed);
 }
+
+int TransactionImpl::Get(const std::string& key, std::string* val)
+{
+  NodeRef getter = src_root_;
+  // Keep looking until we're Nil (We've reached the end)
+  while (getter != Node::Nil())
+  {
+    if (getter->key() == key)
+    {
+      *val = getter->val();
+      return EXIT_SUCCESS;
```

Comment:
Use `0` instead of `EXIT_SUCCESS`. While `EXIT_SUCCESS` is usually zero, it is platform dependent and typically used by a process that is exiting.